### PR TITLE
Leaderboard: hit the API

### PR DESCRIPTION
**Pull request by Bupilipili**
### In this branch

-Created a new game with the name of your choice by using the API.
-Make sure that you saved the ID of your game that the API will return.
-Implemented the "Refresh" button that will get all scores for a game created by you from the API (receiving data from the API and parsing the JSON).
-Implemented the form "Submit" button that will save a score for a game created by you (sending data to the API).
-Use async and await JavaScript features to consume the API.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,6 +1,7 @@
+/* eslint-disable no-unused-vars */
+
```
Comment (V-Blaze) [inline src/index.js]: - You have done great so far. :+1: :+1: Saving and fetching the scores from an API works great. But there are some changes you need to make before moving to the next milestone. :+1: :+1: 

- [ ] Following the best practices, you are not meant to use any linters diable rule in your codebase. Linters are there to make sure your code follows the best standard, I'll suggest you remove this and fix the error. :+1:

File: src/modules/classCollection.js [hunk 2]
```diff
@@ -1,41 +1,124 @@
 /* eslint-disable import/prefer-default-export */
+/* eslint-disable no-console */
```
Comment (V-Blaze) [inline src/modules/classCollection.js]: - [ ] As I pointed out above, As required by Microverse and Following the best practices, you are not meant to use any linters diable rule in your codebase. Linters are there to make sure your code follows the best standard, I'll suggest you remove this and fix the error. :+1:

File: src/modules/classRecord.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
 /* eslint-disable import/prefer-default-export */
```
Comment (V-Blaze) [inline src/modules/classRecord.js]: - [ ] As I pointed out above, As required by Microverse and Following the best practices, you are not meant to use any linters diable rule in your codebase. Linters are there to make sure your code follows the best standard, I'll suggest you remove this and fix the error. :+1:

File: src/style.css [hunk 4]
```diff
@@ -115,7 +115,7 @@ body {
   box-shadow: 4px 3px 2px #000;
```
Comment (V-Blaze) [inline src/style.css:107]: - [OPITONAL] You did great styling your project. :+1: :+1: But I'll suggest you remove the red background color that flashes anytime a user moves the mouse into the app, its not subtle to the eye and makes looking at the UI a bit stain. :+1: :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @Bupilipili , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Posting and fetching from API works great✔
-  Used async-await pattern👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by roshan-bajgain — APPROVED** [review]
Hi @Bupilipili, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- No linter errors. ✔️
- Good pull request and title ✔️ 
- Good job on implementing the changes ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**